extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/channels.xml

```diff
@@ -37,6 +38,13 @@
 		<state readOnly="true" pattern="%.2f °" />
 	</channel-type>
 
+	<channel-type id="shadeLength">
+		<item-type>Number</item-type>
+		<label>Shade Length</label>
```

Comment:
What is the unit of the length? Since the astro binding does not make use of the UOM feature it would be nice to have at least the unit in the label. I assume it is "meter"?

Reply:
Since the description says "ratio" this might actually be only a scalar value. @clinique do you have a link (wikipedia, etc.) which describes this number?

Reply:
Yes, it is only a ratio. So, depending upon the object height and unit the ratio gives the appropriate shade length.

http://gsp.humboldt.edu/olm_2015/Courses/GSP_216_Online/lesson2-2/photogrammetry.html

Reply:
Calculations controlled with https://www.suncalc.org/

Reply:
And wikipedia : https://en.wikipedia.org/wiki/Shadow

Reply:
Ah "ratio", that explains thing... :) Actually I ignored the description so I did not see that is is a ratio. 

So can you please add "Ratio" to the label as well. In PaperUI for example you only see the label and might wonder which unit it is if the term "Ratio" is missing.